fly.toml

```diff
@@ -0,0 +1,56 @@
+# fly.toml file generated for hotel-app on 2022-12-21T23:01:11+03:00
+
+app = "hotel-app"
+kill_signal = "SIGINT"
+kill_timeout = 5
+processes = []
+
+[build]
+  [build.args]
+    BUILD_COMMAND = "bin/rails fly:build"
+    SERVER_COMMAND = "bin/rails fly:server"
+    PROD_PACKAGES = "git build-essential libpq-dev wget vim curl gzip xz-utils libsqlite3-dev libvips libvips-dev libvips-tools libglib2.0-dev glib-2.0-dev libpng-dev libjpeg-dev"
+[deploy]
+  release_command = "bin/rails fly:release"
+
+[env]
+  PORT = "8080"
+
+[experimental]
+  allowed_public_ports = []
+  auto_rollback = true
+
+[[services]]
+  http_checks = []
+  internal_port = 8080
+  processes = ["app"]
+  protocol = "tcp"
+  script_checks = []
+  [services.concurrency]
+    hard_limit = 25
+    soft_limit = 20
+    type = "connections"
+
+  [[services.ports]]
+    force_https = true
+    handlers = ["http"]
+    port = 80
+
+  [[services.ports]]
+    handlers = ["tls", "http"]
+    port = 443
+
+  [[services.tcp_checks]]
+    grace_period = "1s"
+    interval = "15s"
+    restart_limit = 0
+    timeout = "2s"
+
+[[statics]]
+  guest_path = "/app/public"
+  url_prefix = "/"
+
+
+[mounts]
+  source="hotel_app"
+  destination="/data"
```

Comment:
Bu kısmı '/storage' olarak değiştirebilirdin. activestorage folderını değiştirmek yerine ama şu an için sorun yok